### PR TITLE
Unload modules fix

### PR DESCRIPTION
**Pull request by Parfyonator**
Added modules unload code to 'deinitialize' method of the 'ProtobufLoader' class. Also removed modules unload from the python client tests.

### CODE REVIEW

File: exonum_client/protobuf_loader.py [hunk 1]
```diff
@@ -161,6 +161,12 @@ def deinitialize(self) -> None:
         sys.path.remove(self._proto_dir)
         shutil.rmtree(self._proto_dir)
 
+        # Unload any previously loaded `exonum_main` modules from test_exonum_client:
```
Comment (popzxc) [inline exonum_client/protobuf_loader.py]: from `test_exonum_client`? Maybe just "Unload any previously loaded protobuf modules"?

File: tests/module_user.py [hunk 2]
```diff
@@ -8,11 +8,7 @@
 
 class ModuleUserTestCase(unittest.TestCase):
```
Comment (popzxc) [inline tests/module_user.py]: Seems like this class is not needed anymore?

File: exonum_client/protobuf_loader.py (hunk 1 above)
Comment (Parfyonator) [reply exonum_client/protobuf_loader.py]: Yes. My bad )

File: tests/module_user.py (hunk 2 above)
Comment (Parfyonator) [reply tests/module_user.py]: I guess so. I was not sure if it should be deleted (I thought maybe you have some plans for this).

Comment (popzxc) [reply tests/module_user.py]: Nope, feel free to remove so.

Comment (Parfyonator) [reply tests/module_user.py]: Done

**Pull request merged**